cfgov/scripts/add_enforcement_data.py

```diff
@@ -0,0 +1,130 @@
+# Need court, institution type, docket number, tags(sp pop&prod), categories
+# status is record type
+
+import csv
+from datetime import datetime as dt
+from decimal import Decimal
+from re import sub
+
+from v1.models.learn_page import (
+    EnforcementActionDisposition, EnforcementActionPage
+)
+
+
+def strip_fields(f, *args):
+    return [f[i].strip() for i in args]
+
+
+def make_date(d):
+    return dt.strptime(d, '%m/%d/%y') if d else ''
+
+
+def money_to_dec(m):
+    s = sub(r'[^\d.]', '', m) or 0
+    return Decimal(s)
+
+
+record_to_status = {
+    'Final Order': 'post-order-post-judgment',
+    'Dismissal': 'expired-terminated-dismissed',
+    '': 'pending-litigation'
+}
+
+cdata = {}
+rdata = {}
+
+
+def add_relief(data_file='./cfgov/scripts/pea.csv'):
+    with open(data_file, 'r', encoding='utf-8-sig') as csv_file:
+        split = csv.reader(csv_file, delimiter=',')
+        # header
+        next(split)
+
+        for fields in split:
+            (name,
+             record_type,
+             date_filed,
+             final_order_date,
+             dismissal_date,
+             total_consumer_relief,
+             civil_money_penalties,
+             url) = strip_fields(fields, 1, 2, 3, 4, 5, 6, 17, 23)
+
+            o = {
+                'name': name,
+                'status': record_to_status[record_type],
+                'date_filed': make_date(date_filed),
+                'final_order_date': make_date(final_order_date),
+                'dismissal_date': make_date(dismissal_date),
+                'total_consumer_relief': money_to_dec(total_consumer_relief),
+                'civil_money_penalties': money_to_dec(civil_money_penalties)
+            }
+
+            try:
+                rdata[url].append(o)
+            except KeyError:
+                rdata[url] = [o]
+
+
+def add_counts(data_file='./cfgov/scripts/pea_counts.csv'):
+    with open(data_file, 'r', encoding='utf-8-sig') as csv_file:
+        split = csv.reader(csv_file, delimiter=',')
+        # header
+        next(split)
+
+        for fields in split:
+            settled, institution_type, url = strip_fields(fields, 2, 3, 5)
+            if 'Individual' in institution_type:
```

Comment:
I see that some lines of the csv have multiple Institution Type values. So, if one says "Non-Bank; Individual", this code will call it "Individual". Is that what the stakeholder wants?

Reply:
This is somewhat inconsistent in the data (not all `Individual`s are `Non-Bank; Individual`s) and this was just a choice I made with the somewhat intermediate data we have now. When we get the full, final, csv of all final_dispositions I will revisit this decision/bounce it off the stakeholder.